dist/examples/request-body.json

```diff
@@ -1,6 +1,6 @@
 {
   "client_params": {
-    "group": "workers",
-    "node_uuid": "c988d250-9fdf-4cdc-bed3-9037c56406fb"
+    "group": "default",
```

Comment:
Any reason changing this? It is still `workers` in #1

Reply:
To make it easy to test/debug a bare configuration with just the default group.
But good catch, I've update the example in #1 so that they are in sync.